flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/JobMaster.java

```diff
@@ -853,6 +855,12 @@ public CompletableFuture<ExecutionGraphInfo> requestJob(Time timeout) {
         return CompletableFuture.completedFuture(schedulerNG.requestJob());
     }
 
+    @Override
+    public CompletableFuture<CompletedCheckpoint> triggerCheckpoint(
```

Comment:
nit: this can be added next to the other `triggerCheckpoint` method

Reply:
There will be no other `triggerCheckpoint` method in this class, that method is implemented in its parent interface so it will not be defined explicitly here.